src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -82,7 +95,17 @@
                                     }
                                     else
                                     {
-                                        <p>@Loc[nameof(Dashboard.Resources.Metrics.MetricsSelectInstrument)]</p>
+                                        @if (ViewportInformation.IsDesktop)
+                                        {
+                                            <p>@Loc[nameof(Dashboard.Resources.Metrics.MetricsSelectInstrument)]</p>
+                                        }
+                                        else
+                                        {
+                                            <a class @* class needed to prevent color from being removed from reboot style *@
+                                               @onclick="@(async () => await _contentLayout!.OpenMobileToolbarAsync())">
+                                                @Loc[nameof(Dashboard.Resources.Metrics.MetricsSelectInstrument)]
+                                            </a>
```

Comment:
The `@* ... *@` comment here seems to be causing a problem that breaks the metrics view.

1. Open Metrics
2. Click "View attributes" (❓confusing text when presented with "Select a resource to view metrics")
3. Open the "Resource" dropdown and select a resource
4. Exception

   > InvalidCharacterError: Failed to execute 'setAttribute' on 'Element': '@* class needed to prevent color from being removed from reboot style *@' is not a valid attribute name.

   Which is thrown at: 

   > Microsoft.AspNetCore.Components.RenderTree.Renderer.InvokeRenderCompletedCallsAfterUpdateDisplayTask(Task updateDisplayTask, Int32[] updatedComponents)

Reply:
Can't have a comment inside an element.